leaflet/leaflet.d.ts

```diff
@@ -5,7 +5,42 @@
 
 /// <reference path="../geojson/geojson.d.ts" />
 
+type NativeMouseEvent = MouseEvent;
+
 declare namespace L {
+    export class Class {
+        static extend(props:any):any/* how to return constructor of self extended type ? */;
+        static include(props:any):any /* how to return self extended type ? */;
+        static mergeOptions(props:any): any /* how to return self extended type ? */;
+        static addInitHook(initHookFn: ()=> void): any/* how to return self extended type ? */;
+    }
+
+    export class DomUtil {
+        static get(id: string|HTMLElement): HTMLElement;
```

Comment:
Looks great, @gorshkov-leonid, the only two things I would change is splitting this `get` method declaration in two, one for `string` and the other for `HTMLElement`, so that we can see it as overloads and making spacing consistent. Thanks in advance.